src/components/shared/page.tsx

```diff
@@ -163,7 +163,7 @@ function ArticleBody({ capi, imageSalt, getAssetLocation }: BodyProps): ElementW
 
     const insertAdPlaceholders = getAdPlaceholderInserter(capi.fields?.shouldHideAdverts ?? false);
 
-    const item = fromCapi(JSDOM.fragment)(capi);
+    const item = fromCapi(JSDOM.fragment.bind(null))(capi);
```

Comment:
What's this for?

Reply:
Consequence of [`this` in JS](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unbound-method.md).